src/backend.org

```diff
@@ -2894,6 +2912,7 @@ class NodalCentre(Entity):
 
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(128), nullable=False)
+    centre_status = db.Column(db.String(128), nullable=False)
```

Comment:
Please remove **centre_status** attribute from this method also from this branch. As this branch is to add only **user_status** changes. If other branch has the same change that code might create a conflict.